api/src/main/java/net/skinsrestorer/api/reflection/ReflectionUtil.java

```diff
@@ -65,6 +65,15 @@ public class ReflectionUtil {
     private ReflectionUtil() {
     }
 
+    public static boolean classExists(String clazz) {
```

Comment:
Consider using a better name
e.g. classStringName

Reply:
Does not make sense to change, as all other method parameters in this class use clazz. It's just shorter. And "class" is a reserved java keyword.